en/firefox_cloud_services_tos.md

```diff
@@ -38,22 +36,20 @@ This top section summarizes the terms below. This summary is provided to help yo
 
     (d) Firefox Notes lets you take notes in Firefox.  A Firefox Account is required to view, edit, and delete your Notes across devices.  
 
-    (e) Firefox Send lets you share encrypted files that automatically expire.  A Firefox Account is required for large file sizes. 
-
-    (f) Firefox Sync lets you access information stored in Firefox across devices, including your tabs, awesome bar, passwords, bookmarks, and browser preferences. You can enable and disable Firefox Sync through the Tools section of Firefox.  A Firefox Account is required. 
+    (e) Firefox Sync lets you access information stored in Firefox across devices, including your tabs, awesome bar, passwords, bookmarks, and browser preferences. You can enable and disable Firefox Sync through the Tools section of Firefox.  A Firefox Account is required. 
```

Comment:
A few notes:
* We dropped the use of Sync with Proton, so it should be lowercase.
* We talk about Settings, not Tools (and this was unified with Proton)